config/kubernetes/sig-release/teams.yaml

```diff
@@ -247,58 +246,52 @@ teams:
         description: Members of the current Release Team and subproject owners.
         members:
         - alejandrox1 # subproject owner
-        - ams0 # 1.21 Bug Triage Shadow
         - annajung # 1.22 RT Lead Shadow
-        - arunmk # 1.21 Enhancements Shadow
         - ashnehete # 1.22 Release Docs Shadow
-        - bai # 1.20 RT Lead Shadow
         - carlisia # 1.22 Release Docs Shadow
-        - celestehorgan # 1.20 Release Notes Shadow
-        - ChandaniM123 # 1.21 Docs Shadow
-        - chris-short # 1.20 Communications Shadow
         - chrisnegus # 1.22 Release Docs Shadow
```

Comment:
Please add @cici37 she is 1.22 Release Notes Shadow

Reply:
thanks for catching that. Fixed!